jscomp/runtime/js_weakmap.ml

```diff
@@ -25,3 +25,13 @@
 (** ES6 WeakMap API *)
 
 type ('k, 'v) t
+
+external make : unit -> ('k, 'v) t = "WeakMap" [@@mel.new]
+external get : key:'k Js.dict -> 'v option = "get" [@@mel.send.pipe: ('k, 'v) t]
```

Comment:
hmmm, I guess this could also take a function reference instead of an object, but foregoing that is probably fine for now, as to avoid complicating the API too.